altinn-client/src/main/java/no/difi/meldingsutveksling/AltinnWsClientFactory.java

```diff
@@ -0,0 +1,68 @@
+package no.difi.meldingsutveksling;
+
+//import com.sun.xml.ws.developer.JAXWSProperties;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)